smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -109,53 +74,15 @@ class OtelSpringStarterSmokeTest {
   @Autowired private OtelResourceProperties otelResourceProperties;
   @Autowired private OtlpExporterProperties otlpExporterProperties;
 
-  @Configuration(proxyBeanMethods = false)
-  static class TestConfiguration {
-
-    @Bean
-    ConfigurableMetricExporterProvider otlpMetricExporterProvider() {
-      return new ConfigurableMetricExporterProvider() {
-        @Override
-        public MetricExporter createExporter(ConfigProperties configProperties) {
-          return METRIC_EXPORTER;
-        }
-
-        @Override
-        public String getName() {
-          return "memory";
-        }
-      };
-    }
-
-    @Bean
-    ConfigurableSpanExporterProvider otlpSpanExporterProvider() {
-      return new ConfigurableSpanExporterProvider() {
-        @Override
-        public SpanExporter createExporter(ConfigProperties configProperties) {
-          return SPAN_EXPORTER;
-        }
+  @Autowired OpenTelemetry openTelemetry;
 
-        @Override
-        public String getName() {
-          return "memory";
-        }
-      };
-    }
-
-    @Bean
-    ConfigurableLogRecordExporterProvider otlpLogRecordExporterProvider() {
-      return new ConfigurableLogRecordExporterProvider() {
-        @Override
-        public LogRecordExporter createExporter(ConfigProperties configProperties) {
-          return LOG_RECORD_EXPORTER;
-        }
+  @BeforeEach
+  void initOpenTelemetry() {
+    testing = new SpringSmokeTestRunner(openTelemetry);
+  }
```

Comment:
as far as I can tell this doesn't really initialize open telemetry

Reply:
good point - I'll rename it in a later PR